react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { activeIndex: 0 };
+    this.revolver = this.revolver.bind(this);
+    this.counter = this.counter.bind(this);
+    this.componentDidMount = this.componentDidMount.bind(this);
```

Comment:
You don't need to `bind` `componentDidMount`, `this` will be correct inside of it since it is defined directly in the class and not passed to something else

Reply:
Thank you for pointing this out, Scott